nodejs/clients.js

```diff
@@ -48,27 +41,27 @@ async function main() {
     { ssn: exampleDocument["ssn"] },
     { $set: exampleDocument },
     { upsert: true }
-  )
+  );
 
   // Performs a read using the encrypted client, querying on an encrypted field
   const csfleFindResult = await csfleClientPatientsColl.findOne({
-    ssn: exampleDocument["ssn"]
-  })
+    ssn: exampleDocument["ssn"],
+  });
   console.log(
     "Document retreived with csfle enabled client:\n",
     csfleFindResult
-  )
+  );
 
   // Performs a read using the regular client. We must query on a field that is
   // not encrypted.
   // Try - query on the ssn field. What is returned?
   const regularFindResult = await regularClientPatientsColl.findOne({
-    name: "Jon Doe"
-  })
-  console.log("Document retreived with regular client:\n", regularFindResult)
+    name: "Jon Doe",
+  });
+  console.log("Document retreived with regular client:\n", regularFindResult);
 
-  await regularClient.close()
-  await csfleClient.close()
+  await regularClient.close();
+  await csfleClient.close();
```

Comment:
Question: should this be called either in `finally`, or perhaps immediately after you're done fetching the results?